### DISCUSSION TITLE
Missing translations in the code list "innovative-acquisition.gc"

### DISCUSSION THREAD

**Discussion by Colan71** [OP-TED/eForms-SDK#298] [Q&A] [answered]
Hi, 
Can you please provide the English translation for proc-innov and prod-innov in the code list innovative-acquisition.gc. They seem to be missing.

![inn_ac1](https://user-images.githubusercontent.com/79446518/212112556-c317e162-4e3d-4cac-9914-82c22405809b.png)
![inn_ac2](https://user-images.githubusercontent.com/79446518/212112566-3abde039-979c-4142-b878-699fb1258a5a.png)

**Comment by Colan71** [comment, marked as answer]
I just spotted that this will be part of the 1.5.1 release!

**Comment by bertrand-lorentz** [comment]
Thanks for your feedback.
Indeed, this was already reported in issue #280 and it will be fixed in SDK 1.5.1.
